cpp/src/Ice/ConnectionI.h

```diff
@@ -348,6 +376,9 @@ namespace Ice
         // The number of user calls currently executed by the thread-pool (servant dispatch, invocation response, ...)
         int _upcallCount;
 
+        // The number of outstanding dispatches, including heartbeat callback "dispatches".
```

Comment:
The implementation says otherwise, `_dispatchCount` isn't increased for heartbeats upcalls.

Reply:
Fixed comment.